content/en/synthetics/api_tests/dns_tests.md

```diff
@@ -0,0 +1,162 @@
+  tag: "Blog"
+  tewt: "DNS monitoring with Datadog"
+---
+
+## Overview
+
+DNS tests allow you to proactively monitor the **resolvability and lookup times of your DNS records using any nameserver**. If resolution is unexpectedly slow or if a DNS server answers with unexpected A, AAAA, CNAME, TXT, or MX entries, Datadog sends you an **alert with details on the failure**, allowing you to quickly pinpoint the root cause of the issue and fix it.
+DNS tests can run from [managed][1] and [private locations][2] allowing you to run tests on **public domains** as well as **internal services** that might be using internal DNS.
+
+## Configuration
+
+After choosing the type of test you are willing to create ([`HTTP`][3], [`SSL`][4], [`TCP`][5], or [`DNS` test][6]), you can define your test's request.
+
+### Define request
+
+{{< img src="synthetics/api_tests/dns_test_config.png" alt="Define DNS query"  style="width:90%;" >}}
+
+1. Specify the **Domain** you want your test to query, e.g., `www.example.com`.
+2. Specify the `DNS Server` to use (optional), it can be a domain name or an IP address. If not specified, your DNS test performs resolution using `8.8.8.8`, with a fallback on `1.1.1.1` and an internal AWS DNS server.
+3. **Name** your DNS test.
+4. Add `env` **Tags** as well as any other tag to your DNS test. You can then use these tags to quickly filter through your Synthetic tests on the [Synthetic Monitoring homepage][7].
+5. Select the **Locations** to run your DNS test from: DNS tests can run from [managed][1] and [private locations][2] depending on whether you are willing to monitor a public or a private domain.
+
+Click on **Test URL** to try out the request configuration. You should see a response preview show up on the right side of your screen.
+
+### Specify test frequency
+
+DNS tests can run:
+
+* **On a schedule** to ensure your most important services are always accessible to your users. Select the frequency you want Datadog to run your DNS test.
+
+{{< img src="synthetics/api_tests/schedule.png" alt="Run API tests on schedule"  style="width:90%;" >}}
+
+* [**Within your CI/CD pipelines**][8].
+* **On-demand** to run your tests whenever makes the most sense for your teams.
+
+### Define assertions
+
+Assertions define what an expected test result is. When hitting `Test URL` basic assertions on `response time` and available records are added. You must define at least one assertion for your test to monitor.
+
+| Type                | Record type                                                     | Operator                                           | Value type                 |
+|---------------------|-----------------------------------------------------------------|----------------------------------------------------|----------------------------|
+| response time       |                                                                 | `is less than`                                     | _Integer (ms)_             |
```

Comment:
@MargotLepizzera Is this supposed to be blank for record type? Just wanted to verify as the other two have information.

Reply:
Yes that's expected - it looks a bit weird 😅 but that column only exists for the `every record` and `at least one record` lines!